content/en/05-use-case-tutorials/network-monitoring.md

```diff
@@ -0,0 +1,134 @@
+- [Visualize the flow of DNS requests within your cluster.](#visualize-dns-requests-made-within-the-cluster)
+- [Visualize TCP drops and TCP retransmits across your cluster.](#visualize-tcp-drops-and-tcp-retransmissions-across-the-cluster)
+
+**Prerequisites**
+
+1. You will need a Kubernetes cluster with Pixie installed. If you do not have a cluster, you can create a minikube cluster and install Pixie using our [install guides](/installing-pixie/quick-start/).
+
+## Live network traffic map
+
+A global view of the network traffic flowing within a cluster can be used to:
+
+- Quickly determine which services are communicating.
+- Identify any misconfigured services.
+- Identify load balancing issues.
+
+Let’s use the `px/net_flow_graph` script to see a graph of all of the network traffic passing through the cluster:
+
+1. Open the [Live UI](http://work.withpixie.ai/) and select `px/net_flow_graph` from the `script` drop-down menu at the top.
+
+> When the script opens, you’ll get an error indicating a value is missing for the `namespace` required argument. The UI denotes required script arguments with an asterisk after the argument name.
+
+::: div image-l
+<svg title='' src='use-case-tutorials/missing_required_arg.png'/>
+:::
+
+2. Enter `pl` for the required `namespace` argument: select the drop down arrow next to the `namespace` argument, type `pl`, and hit enter.  `pl` is the namespace that Pixie deploys to.
+
+::: div image-xl relative
+<PoiTooltip top={25} left={15}>
+<strong>Click anywhere on the graph to interact</strong>
+{': '}
+pan, zoom, or rearrange individual nodes.
+</PoiTooltip>
+
+<PoiTooltip top={37} left={62}>
+<strong>Hover over an Edge</strong>
+{' '}
+for network transmission stats. Thicker lines indicate more traffic.
+</PoiTooltip>
+
+<PoiTooltip top={60} left={87}>
+<strong>Enable Hierarchy View</strong>
+{' '}
+for a different view of the graph.
+</PoiTooltip>
+
+<svg title='' src='use-case-tutorials/net_flow_graph.png'/>
+:::
+
+> This script shows a mapping of all the outgoing connections for the pods in the specified namespace.
+
+> Grey hexagonal icons represent pods making network requests. Blue circles represent remote endpoint. If a request’s remote endpoint is within the cluster, then the IP address is resolved to a pod/service name.
+
+> Let's filter the graph to only show communication to the `pl-nats` pod.
+
+3. Select drop down arrow next to the `to_entity_filter` argument, type `pl-nats`, and press enter to re-run the script.
+
+> The graph should update to only show network traffic sent to the `pl-nats` pod. Pixie uses NATS as our messaging system.
+
+4. Clear the `to_entity_filter` value by selecting the drop-down arrow and pressing enter.
+
+5. Scroll down to the table below the graph. This table contains the same data that is used to construct the graph above.
+
+## Live DNS request map
+
+A global view of DNS requests made within a cluster can be used to:
```

Comment:
I would rephrase so it doesn't sound canned w/the above similar language.

Another capability Pixie provides is the ability to inspect and analyze DNS traffic. Here are somethings Pixie can help you do: